### PR TITLE
[activity] add discord.Spotify.track_url

### PR DESCRIPTION
**Pull request by quiprr**
## Summary

Add a new property to discord.Spotify that reflects the track URL of the currently playing song.

## Checklist

- [x] If code changes were made then they have been tested.
    - [ ] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

File: discord/activity.py [hunk 1]
```diff
@@ -674,6 +674,11 @@ def album_cover_url(self):
     def track_id(self):
         """:class:`str`: The track ID used by Spotify to identify this song."""
         return self._sync_id
+    
+    @property
+    def track_url(self)
```
Comment (Dorukyum) [inline discord/activity.py:679]: You forgot a colon.

**Review by Dorukyum — CHANGES_REQUESTED** [review]

File: discord/activity.py (hunk 1 above)
Comment (quiprr) [reply discord/activity.py]: Fixed, thank you

File: discord/activity.py [hunk 2]
```diff
@@ -674,6 +674,11 @@ def album_cover_url(self):
     def track_id(self):
         """:class:`str`: The track ID used by Spotify to identify this song."""
         return self._sync_id
+    
+    @property
+    def track_url(self):
+        """:class:`str`: The track URL to listen on Spotify."""
```
Comment (Zomatree) [inline discord/activity.py:680]:
```suggestion
        """:class:`str`: The track URL to listen on Spotify.
        
        .. versionadded:: 2.0
        """
```

File: discord/activity.py [hunk 3]
```diff
@@ -674,6 +674,11 @@ def album_cover_url(self):
...
+        return 'https://open.spotify.com/track/' + self.track_id
```
Comment (Zomatree) [inline discord/activity.py:681]:
```suggestion
        return f'https://open.spotify.com/track/{self.track_id}'
```

Comment (quiprr) [reply discord/activity.py:684]: Following syntax of line671 for uniformity.

File: discord/activity.py [hunk 4]
```diff
@@ -674,6 +674,14 @@ def album_cover_url(self):
...
         """:class:`str`: The track ID used by Spotify to identify this song."""
         return self._sync_id
+    
+    @property
+    def track_url(self):
+        """:class:`str`: The track URL to listen on Spotify.
+        
+        .. versionadded:: 2.0
+        """
+        return 'https://open.spotify.com/track/' + self.track_id
```
Comment (Rapptz) [inline discord/activity.py:684]:
```suggestion
        return f'https://open.spotify.com/track/{self.track_id}'
```

File: discord/activity.py [hunk 5]
```diff
@@ -674,6 +674,14 @@ def album_cover_url(self):
     def track_id(self):
         """:class:`str`: The track ID used by Spotify to identify this song."""
         return self._sync_id
+    
+    @property
+    def track_url(self):
```
Comment (Rapptz) [inline discord/activity.py:679]:
```suggestion
    def track_url(self) -> str:
```

**Review by Rapptz — APPROVED** [review]

**Pull request merged**